### PR TITLE
welcome page 7

### PR DESCRIPTION
**Pull request by Errolyn**
## Description

Add welcome page based on whatsnew 74

## Issue / Bugzilla link
#8574

## Testing
There is one coded block for FR, DE, en- and everything else. They should all work.
http://localhost:8000/firefox/welcome/7/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/welcome/page7.html [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+{% call hero(
+    title=_('It’s okay to like Facebook'),
+    desc=_('If you still kinda like Facebook but don’t trust them, then try the Facebook Container extension by Firefox and make it harder for them to track you around the web.'),
+    class='mzp-t-firefox mzp-t-dark',
+    include_cta=True,
+    heading_level=1
+  ) %}
+
+  <p class="primary-cta">
+    <a href="https://addons.mozilla.org/firefox/addon/facebook-container/?src=external-www.mozilla.org-whatsnew74&utm_source=www.mozilla.org-whatsnew74&utm_medium=referral&utm_campaign=feb2020" class="mzp-c-button mzp-t-product" id="facebook-container-button" rel="external">{{ _('Get Facebook Container') }}</a>
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/welcome/page7.html]: These URL params still mention whatsnew74 and need to be updated for this page, here and on the other links. @hoosteeno or @sghoseWI might have input on what the preferred values should be.

File: media/css/firefox/welcome.scss [hunk 2]
```diff
@@ -71,14 +71,15 @@ $image-path: '/media/protocol/img';
     }
 }
 
-.mzp-c-hero-title {
+.mzp-c-hero-title, .hero-subheading {
```
Comment (craigcook) [inline media/css/firefox/welcome.scss]: I don't think the `.hero-subheading` class is used. Is this is a leftover?

**Comment by craigcook** [comment]
Also don't forget to update that string on welcome 6!

File: media/css/firefox/welcome.scss (hunk 2 above)
Comment (Errolyn) [reply media/css/firefox/welcome.scss]: yep you are totally right, I was trying something and missed taking that bit out!

**Pull request merged**